src/common/utils/gpo.h

```diff
@@ -446,4 +447,9 @@ namespace powertoys_gpo {
             return getConfiguredValue(POLICY_CONFIGURE_ENABLED_POWER_LAUNCHER_ALL_PLUGINS);
         }        
     }
+
+    inline gpo_rule_configured_t getConfiguredQoiThumbnailsEnabledValue()
+    {
+        return getConfiguredValue(POLICY_CONFIGURE_ENABLED_QOI_THUMBNAILS);
```

Comment:
This is a utility policy. You need  `getUtilityEnabledValue`.


```suggestion
        return getUtilityEnabledValue(POLICY_CONFIGURE_ENABLED_QOI_THUMBNAILS);
```